### PR TITLE
#163745936 Chore update offices controller 

### PR DESCRIPTION
**Pull request by ibkadeeko**
#### What does this PR do?
Update offices controller to persist data

#### Description of Task to be completed?
Rewrite offices controller to query database and persist data
Check that all endpoints on offices routes are working properly
write unit tests for all endpoints 

#### How should this be manually tested?
- First clone the project on your local computer. 
- To do that create a folder in your local machine, open the terminal in that folder and run `https://github.com/ibkadeeko/Politico.git`
- cd into the project on your terminal and run `npm install` to install all dependencies
- start the server by running `npm start`

Using Postman test every endpoint above
- Make POST and GET requests to http://localhost:5000/api/v1/offices using values:

Using Mocha run `npm test`

#### What are the relevant pivotal tracker stories?
#163745936

### CODE REVIEW

File: server/controllers/offices.js [hunk 1]
```diff
@@ -1,58 +1,72 @@
 import officesDb from '../models/offices';
```
Comment (tutugodfrey) [inline server/controllers/offices.js:1]: Where are you using officesDb? Remove it if you are not using it.

File: server/test/parties.test.js [hunk 2]
```diff
@@ -188,3 +188,16 @@ describe('Parties', () => {
       });
   });
 });
+
+describe('Root Route', () => {
```
Comment (tutugodfrey) [inline server/test/parties.test.js:192]: This test does not have anything to do with parties. Why do we have it here?

File: server/controllers/offices.js (hunk 1 above)
Comment (ibkadeeko) [reply server/controllers/offices.js:1]: Ok

File: server/test/parties.test.js (hunk 2 above)
Comment (ibkadeeko) [reply server/test/parties.test.js:192]: Its a test for the root route that covers both offices and parties. I had put it in parties.test.js earlier.

**Review by tutugodfrey — APPROVED** [review]

**Pull request merged**